repo.go

```diff
@@ -319,10 +295,6 @@ func (r *Repo) timestamp() (*data.Timestamp, error) {
 }
 
 func (r *Repo) ChangePassphrase(keyRole string) error {
-	if !verify.ValidRole(keyRole) {
```

Comment:
Maybe `!roles.IsTopLevelRole(keyRole)`? If I recall correctly, so far it's possible to generate a key (ergo have a keys file that can be passphrase encrypted) only for top-level roles.

Reply:
Fixed.